test/Makefile

```diff
@@ -1,3 +0,0 @@
-
-test_message_format: 
-	g++ -I .. test_message_format.cc ../build/carta-protobuf/libcarta-protobuf.a -l protobuf
```

Comment:
Did this and the message format test get deleted by mistake? We should probably convert them to the same kind of test for consistency. Did we ever decide on a clean way to build tests with dependencies on the rest of the CARTA code without rebuilding it in all the individual subdirectories? Possibly a single test build with some kind of top-level suite that can run any subset of the tests?

Reply:
it wasn't entirely by accident, but I removed it because it seemed like an inconsistent test that should probably be updated to a proper unit test if its still required.

Reply:
perhaps I should leave that cleanup for another PR :smile: